source/blog/2016-06-14-rspec-3.5-has-been-released.md

```diff
@@ -0,0 +1,125 @@
+---
+title: RSpec 3.5 has been released!
+author: Sam Phippen
+---
+
+RSpec 3.5 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
+upgrade for anyone already using any RSpec 3 release, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over 450 commits and 150
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Rails: Support for Rails 5
+
+The headline here is that RSpec 3.5.0 is compatible with Rails 5. As Rails 5
+betas and release candidates have been released, we've been releasing betas of
+3.5.0 to keep up alongside Rails. Due to this being a major release of Rails,
+but not a major release of RSpec, Rails has removed some APIs that we consume.
+This only gets exposed to you, our users, in one place: controller testing.
+
+In [Rails 5](https://github.com/rails/rails/issues/18950) `assigns` and
+`assert_template` are "soft deprecated". Controller tests themselves *are not*,
+and adding `:type => :controller` to your specs is still 100% supported.
+Through Rails 3 and 4: it was both prevalent and idiomatic to use `assigns` in
+controller specs. As this is a minor release of RSpec our commitment to SemVer
+means that we are not going to break your existing controller specs. For
+existing Rails applications that make heavy use of `assigns` adding the
+[`rails-controller-testing`](https://github.com/rails/rails-controller-testing)
+to one's Gemfile will restore `assigns` and `assert_template`. RSpec integrates
```

Comment:
While I appreciate the gender-neutral "one's" here, I think it reads kinda awkwardly and it reads better IMO to rephrase the sentence to use "your" (and is just as gender-neutral):

> If your existing Rails applications makes heavy use of `assigns`, just add the
> [`rails-controller-testing`](https://github.com/rails/rails-controller-testing) gem
> to your Gemfile to restore `assigns` and `assert_template`.